soc/microchip/pic32cxsg/common/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# Makefile - Microchip PIC32CXSG MCU family
+#
+# Copyright (c) 2024 Microchip
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_sources(soc_port.c)
+
+zephyr_sources_ifdef(CONFIG_BOOTLOADER_BOSSA bossa.c)
```

Comment:
This should be dropped for now. When you add on a board (another PR) you can add here.